### PR TITLE
Misc doc changes

### PR DESCRIPTION
**Pull request by kianmeng**
Besides other documentation changes, this commit ensures the generated
HTML doc for HexDocs.pm will become the source of truth for this Elixir
library and leverage on latest features of ExDoc.

### CODE REVIEW

**Comment by kianmeng** [comment]
@edgurgel Any updates on this?

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-Copyright (c) 2013-2014 Eduardo Gurgel Pinho
+# The MIT License
+
+Copyright (c) 2013 Eduardo Gurgel <eduardo@gurgel.me>
```
Comment (nathany-copia) [inline LICENSE.md:3]: I'm not sure about changing the copyright line?

File: README.md [hunk 2]
```diff
@@ -1,49 +1,56 @@
...
-* [ ] /deflate Returns deflate-encoded data.
-* [ ] /digest-auth/:qop/:user/:passwd Challenges HTTP Digest Auth.
+* [ ] `/deflate` Returns deflate-encoded data.
+* [ ] `/digest-auth/:qop/:user/:passwd` Challenges HTTP Digest Auth.
 
-## License
+## Copyright and License
 
-    Copyright 2013-2016 Eduardo Gurgel <eduardo@gurgel.me>
+Copyright (c) 2013 Eduardo Gurgel <eduardo@gurgel.me>
```
Comment (nathany-copia) [inline README.md:53]: same question

**Comment by nathany-copia** [comment]
The readme changes can be viewed at:
https://github.com/kianmeng/httparrot/tree/misc-doc-changes?tab=readme-ov-file

The Travis CI build status is broken both in this PR and on main, but that could be resolved in a separate pull request.

File: LICENSE.md (hunk 1 above)
Comment (kianmeng) [reply LICENSE.md:3]: @nathany-copia IANAL. If I remember correctly, for most cases, using a single year (the year of initial creation) is sufficient and the recommended approach as we don't have to constantly update the latest copyright year.

**Review by edgurgel — APPROVED** [review]

**Comment by edgurgel** [comment]
It took some time but I'm back :laughing: Sorry for the wait, team

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**